packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -10,46 +10,46 @@ This guide covers how to upgrade the Okta Sign-In Widget, which depends on wheth
+After you've completed the widget upgrade, review [Okta Identity Engine overview](/docs/guides/oie-intro/) to take advantage of new features in Identity Engine.
 
 ## Best practice for widget maintenance
 
 For best practices, keep your [widget](https://github.com/okta/okta-signin-widget/blob/master/README.md) up to date. This maintenance is essential so that you can use the latest Identity Engine features and benefit from ongoing improvements to the codebase.
 
 The specific steps to upgrade your widget depend on your [user authentication deployment model](/docs/concepts/redirect-vs-embedded/), which can be one of the following:
 
-* [Redirect authentication](/docs/guides/oie-upgrade-sign-in-widget/main/#upgrade-process-for-a-redirect-sign-in): Okta-hosted with no custom URL domain
-* [Redirect authentication](/docs/guides/oie-upgrade-sign-in-widget/main/#upgrade-process-for-a-redirect-sign-in): Okta-hosted with custom URL domain
-* [Embedded authentication](/docs/guides/oie-upgrade-sign-in-widget/main/#upgrade-process-for-an-embedded-sign-in-widget): Self-hosted. The embedded widget is able to perform the OIDC flow and return OAuth tokens directly within the application.
+* [Redirect authentication](#upgrade-process-for-a-redirect-sign-in-flow): Okta-hosted with no custom URL domain
+* [Redirect authentication](#upgrade-process-for-a-redirect-sign-in-flow): Okta-hosted with custom URL domain
+* [Embedded authentication](#upgrade-process-for-an-embedded-widget): Self-hosted. The embedded widget is able to perform the OIDC flow and return OAuth tokens directly within the application.
 
 ## Best practice for user experience and application policies
+<!-- REMOVE THIS SECTION?? doesn't make sense and links to deprecated file, plus what about Interaction Code flow? -->
+In Classic Engine, the widget had the default option to initialize without application context. In Identity Engine, you can use one of the following OIDC flows to optimize the user's sign-in experience and apply specific application policies:
 
-In Classic Engine, the widget had the default option to initialize without application context. In Identity Engine, you should use one of these OIDC flows so that you can optimize the user's sign-in experience and apply specific application policies:
-
-* [Server-side web application using Authorization Code flow](https://developer.okta.com/code/javascript/okta_sign-in_widget/#server-side-web-application-using-authorization-code-flow)
-* [SPA or native application using PKCE](https://developer.okta.com/code/javascript/okta_sign-in_widget/#spa-or-native-application-using-pkce)
+* [Server-side web application using Authorization Code flow](/code/javascript/okta_sign-in_widget/#server-side-web-application-using-authorization-code-flow)
+* [SPA or native application using PKCE](/code/javascript/okta_sign-in_widget/#spa-or-native-application-using-pkce)
 
 ## Upgrade process for a redirect sign-in flow
```

Comment:
I can't leave a comment at the exact spot, but there is no Customization > Custom Sign In tab in the Admin Console. They must have rearranged the menus again. You'll need to update this section with the correct instructions.